conductor_api/src/conductor/base.rs

```diff
@@ -643,17 +641,24 @@ impl Conductor {
                 }
 
                 if config.logger.logger_type == "debug" {
-                    context_builder = context_builder.with_logger(Arc::new(Mutex::new(
+                    let _logger = FastLoggerBuilder::new()
+                        .set_level_from_str("Trace")
+                        .add_rule_filter(RuleFilter::new("Abort", false, "Red"))
+                        .build()
+                        .expect("Fail to instanciate the logging factory.");
```

Comment:
I don't think we can instantiate the logger here as this would
1. instantiate a new logger every time an instance gets instantiated
2. we don't have any logger before the first instance is alive.

Also we need to pass the logger config over from the conductor config into the builder..

Reply:
Well, also this logger instance gets dropped immediately which explains the errors in app-spec and cluster test.